helpers/content.go

```diff
@@ -35,10 +35,10 @@ import (
 	"sync"
 )
 
-// Length of the summary that Hugo extracts from a content.
+// SummaryLength of the summary that Hugo extracts from a content.
 var SummaryLength = 70
 
-// Custom divider <!--more--> let's user define where summarization ends.
+// SummaryDivider Custom divider <!--more--> let's user define where summarization ends.
```

Comment:
SummaryDivider denotes where content summarization should end. The default is "<!--more-->".